pwnlib/context/__init__.py

```diff
@@ -1071,6 +1072,32 @@ def log_console(self, stream):
             stream = open(stream, 'wt')
         return stream
 
+    @_validator
+    def local_libcdb(self, path):
+        """ 
+        Sets path to local libc-database, get more information for libc-database:
+        https://github.com/niklasb/libc-database
+
+        Works in :attr:`pwnlib.libcdb` when searching by local database provider.
+
+        The default value is ``/var/lib/libc-database``.
+
+        Sets `context.local_libcdb` to empty string or `None` will turn off local libc-database integration.
+
+        Examples:
+
+            >>> context.local_libcdb = '/home/runner/work/pwntools/pwntools/pwnlib/data/elf/libcdb'
```

Comment:
To make this work when running the test locally please look at the other `__init__.py` files in the pwnlib/data/elf/* folders and use `pwnlib.data.elf.libcdb.path` instead of hardcoding the path.